src/process/receipts.rs

```diff
@@ -0,0 +1,297 @@
+use std::collections::HashMap;
+use std::convert::TryFrom;
+
+use diesel::r2d2::{ConnectionManager, Pool};
+use diesel::{ExpressionMethods, JoinOnDsl, PgConnection, QueryDsl};
+use futures::join;
+use num_traits::cast::FromPrimitive;
+use tokio_diesel::AsyncRunQueryDsl;
+use tracing::{error, warn};
+
+use crate::models;
+use crate::schema;
+
+/// Saves receipts to database
+pub(crate) async fn process_receipts(
+    pool: &Pool<ConnectionManager<PgConnection>>,
+    receipts: Vec<&near_indexer::near_primitives::views::ReceiptView>,
+    block_height: u64,
+) {
+    let mut skipping_receipt_ids: Vec<near_indexer::near_primitives::hash::CryptoHash> = vec![];
```

Comment:
It is better to use HashSet given that we want to check for values to be found in it

```suggestion
    let mut skipping_receipt_ids = std::collections::HashSet::<near_indexer::near_primitives::hash::CryptoHash>::new();
```